spec/javascripts/actions/story_spec.js

```diff
@@ -283,11 +285,12 @@ describe('Story Actions', () => {
 
       await Story.dragDropStory(story.id, story.projectId, {
         position: 3.54,
-      })(fakeDispatch, fakeGetState, { Story: FakeStory });
+      }, from)(fakeDispatch, fakeGetState, { Story: FakeStory });
+
+      await wait(300);
```

Comment:
We shouldn't have to wait it in  the test :thinking: , why was this necessary?

Reply:
Probably because I wanted simulate exactly the same behavior of real action. But this isn't really necessary. Thank you again!